mobile app broken

```
mobile.js:141 Error: [$injector:unpr] http://errors.angularjs.org/1.6.6/$injector/unpr?p0=gmfLayersUrlProvider%20…C-%20gmfEditFeature%20%3C-%20gmfObjectEditingManager%20%3C-%20gmfPermalink
```

--- a/contribs/gmf/apps/mobile/index.html
+++ b/contribs/gmf/apps/mobile/index.html
@@ -202,6 +202,7 @@
         module.constant('authenticationBaseUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi');
         module.constant('fulltextsearchUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi/fulltextsearch?limit=20&interface=mobile');
         module.constant('gmfTreeUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi/themes?version=2&background=background&interface=mobile');
+        module.constant('gmfLayersUrl', 'https://geomapfish-demo.camptocamp.net/2.2/wsgi/layers/');
         module.constant('gmfRasterUrl', 'https://geomapfish-demo.camptocamp.net/2.1/wsgi/raster');
         module.constant('gmfShortenerCreateUrl', '');
         module.constant('gmfSearchGroups', []);

